packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,113 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Returns an easily parsable date format string.
+   * @param {string} locale - The locale variant from dateFormatOrder.
+   */
+  static getDateFormat(locale) {
```

Comment:
Perhaps, rename this to getDateFormatVariant or the like. The parameter passed in should be the format order instead of the locale so perhaps rename it something other than locale.

Reply:
I updated the translation files to include the actual variant instead of a number. This should be less confusing, and it also means that the above util method has been removed! :)